experiment/evaluator.py

```diff
@@ -217,7 +292,33 @@ def _parse_libfuzzer_logs(self, log_handle) -> tuple[int, int, bool]:
         crashes = True
         continue
 
-    return cov_pcs, total_pcs, crashes
+    initcov, donecov, lastround = self._parse_fuzz_cov_info_from_libfuzzer_logs(
+        lines)
+
+    # NOTE: Crashes from incorrect drivers will not be counted.
+
+    if crashes:
+      # FP case 1: driver crashes at init or first few rounds.
+      if lastround is None or lastround <= EARLY_FUZZING_ROUND_THRESHOLD:
+        # No cov line has been identified or only INITED round has been passed.
+        # This is very likely the false positive cases.
+        return cov_pcs, total_pcs, False, True, 'FP_CRASH_NEAR_INIT'
+
+      # FP case 2: 1st func of the 1st thread stack is in driver.
+      crash_stacks = self._parse_stacks_from_libfuzzer_logs(lines)
+      for stack_frame in crash_stacks[:1]:
+        if self._stack_func_is_of_testing_project(stack_frame):
+          if 'LLVMFuzzerTestOneInput' in stack_frame:
+            return cov_pcs, total_pcs, False, True, 'FP_CRASH_IN_DRIVER'
```

Comment:
Could we replace these with Python enums? https://docs.python.org/3/library/enum.html 

@DonggeLiu

Reply:
Ah, yes good point. Thanks!